packages/code-du-travail-nlp/Dockerfile

```diff
@@ -24,7 +26,11 @@ ENV FLASK_APP api
 ENV FLASK_APP="development"
 
 RUN python -m venv venv
+
+# hadolint ignore=SC1091
```

Comment:
what does it means ? 
should we fix that in a near future ?

Reply:
this happens when the linter cant find the mentioned source file

im not sure we can "fix" it 🤔 

https://github.com/koalaman/shellcheck/wiki/SC1091